packages/calcite-components/src/components/table/table.tsx

```diff
@@ -0,0 +1,524 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop,
+  State,
+  VNode,
+  Watch,
+} from "@stencil/core";
+import { Scale, SelectionMode } from "../interfaces";
+import {
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent,
+} from "../../utils/loadable";
+import {
+  connectMessages,
+  disconnectMessages,
+  setUpMessages,
+  T9nComponent,
+  updateMessages,
+} from "../../utils/t9n";
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  numberStringFormatter,
+  NumberingSystem,
+} from "../../utils/locale";
+import { TableLayout, TableRowFocusEvent } from "./interfaces";
+import { CSS, SLOTS } from "./resources";
+import { TableMessages } from "./assets/table/t9n";
+import { getUserAgentString } from "../../utils/browser";
+
+/**
+ * @slot - A slot for adding `calcite-table-row` or nested `calcite-table` elements. Content placed here will be rendered in a `table-body` tag.
+ * @slot table-head- A slot for adding `calcite-table-row` and nested `calcite-table-header` elements.
+ * @slot table-foot- A slot for adding `calcite-table-row` and nested `calcite-table-header` elements.
+ * @slot selection-actions - A slot for adding a `calcite-action` or other element to display when `selectionMode !== "none"` and `calcite-table-row` are selected.
```

Comment:
WDYT about modifying the `selectionMode !== "none"` to:

```
 ...when `selectionMode` is not `"none"` and...
```

Reply:
yep, updated this text a bit and included that.